sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Models/Sku.cs

```diff
@@ -51,24 +47,31 @@ public Sku(string tier = default(string), string name = default(string), int? ca
         partial void CustomInit();
 
         /// <summary>
-        /// Gets or sets the service tier
-        /// </summary>
-        [JsonProperty(PropertyName = "tier")]
-        public string Tier { get; set; }
-
-        /// <summary>
-        /// Gets or sets the SKU name
+        /// Gets or sets the name of the SKU, typically, a letter + Number
+        /// code, e.g. P3.
         /// </summary>
         [JsonProperty(PropertyName = "name")]
         public string Name { get; set; }
 
         /// <summary>
-        /// Gets or sets if the SKU supports scale out/in then the capacity
-        /// integer should be included. If scale out/in is not possible for the
-        /// resource this may be omitted.
+        /// Gets or sets the tier or edition of the particular SKU, e.g. Basic,
+        /// Premium.
         /// </summary>
-        [JsonProperty(PropertyName = "capacity")]
-        public int? Capacity { get; set; }
```

Comment:
this will result in a breaking change, please revert or have the breaking change team add the approval label to the spec pr

Reply:
reverted